Sources/Sentry/Public/SentryClient.h

```diff
@@ -41,6 +41,9 @@ SENTRY_NO_INIT
 - (SentryId *)captureEvent:(SentryEvent *)event
                  withScope:(SentryScope *)scope NS_SWIFT_NAME(capture(event:scope:));
 
+- (SentryId *)captureTransaction:(SentryTransaction *)transaction
```

Comment:
missing the `withScope`

Reply:
well, every other method does, `captureEvent` requires a Scope, and right now `captureTransaction` is instantiating a new Scope every time, so either we take a Scope param or pass null, so we save the allocation if not using it at all.

Reply:
> well, every other method does,

It's not really a reason

> captureEvent requires a Scope, and right now captureTransaction is instantiating a new Scope every time,

This is a good point.

@brustolin Can we pass `nil` instead of creating a scope if we're not using it?

> so either we take a Scope param or pass null, so we save the allocation if not using it at all.

Yes, this ^ please

Reply:
For now, it does not accept null in the parameter, and it uses the parameter in a ton of other calls. I guess I could change it all until the sendEvent, but I would have to add some extra flow to verify if scope is nil.

Reply:
OK lets merge this and follow up with that on a next PR.